datafusion/core/tests/sql/information_schema.rs

```diff
@@ -307,14 +307,8 @@ async fn information_schema_show_columns() {
     let result = plan_and_collect(&ctx, "SHOW columns from t").await.unwrap();
     assert_batches_sorted_eq!(expected, &result);
 
-    // This isn't ideal but it is consistent behavior for `SELECT * from T`
-    let err = plan_and_collect(&ctx, "SHOW columns from T")
```

Comment:
This test appears to have been invalid because it treats unquoted table names as case-sensitive

Reply:
Maybe this change would remain more faithful to the original test's stated intent (aka use `"T"` to not normalize the table?

```sql
    let err = plan_and_collect(&ctx, "SHOW columns from \"T\"")
```

Reply:
Good point! Updated!